lib/models/device.coffee

```diff
@@ -249,6 +249,41 @@ exports.getApplicationName = (uuid, callback) ->
 	exports.get(uuid).get('application_name').nodeify(callback)
 
 ###*
+# @summary Get application container id
```

Comment:
I think this summary is not accurate. The `containerId` is one of the properties that can be retrieved from this endpoint.

The summary should be more generic and in line with the function name (e.g: "Get application container information"), or the function should be edited to only return the container id, and be renamed to something like `getContainedId()`.